src/bundle/Resources/public/scss/fieldType/preview/_image-preview.scss

```diff
@@ -0,0 +1,51 @@
+@mixin image-preview() {
```

Comment:
For now, we were adding mixins into the `_mixins.scss` but this file might be big in the future.
What about creating a folder called `mixins` and put them there?
Also, I would suggest importing this mixin in the `ezplatform.scss`, thanks to this mixin are "globally"